### PR TITLE
Added images for dark theme + special asset use case

### PR DESCRIPTION
**Pull request by dev-andreavendrame**
This PR has the aim of:
- Add support for both dark and light Github theme
- Add a special use case for the MultiAsset standard. This is achieved by adding the same asset to multiple tokens, so there is a relation **one to many** between an asset and NFTs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,7 +30,9 @@ After starting their travels, Master and Alice decided to separate and they had
     );
 ```
 
-![alt text](images/MA_1.png)
+![alt text](images/dark_theme/MA_1.png#gh-dark-mode-only)
+![alt text](images/light_theme/MA_1.png#gh-light-mode-only)
```
Comment (ThunderDeliverer) [inline README.md]: This has to be adapted in order to selectively show the desired picture based on the theme (like in the [documentation](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#specifying-the-theme-an-image-is-shown-to)):

```suggestion
<picture>
  <source media="(prefers-color-scheme: dark)" srcset="images/dark_theme/MA_1.png">
  <source media="(prefers-color-scheme: light)" srcset="images/light_theme/MA_1.png">
  <img alt="Alternative text." src="images/light_theme/MA_1.png">
</picture>
```

Currently this is using the legacy way of selecting the image based on theme:

> The old method of specifying images based on the theme, by using a fragment appended to the URL (#gh-dark-mode-only or #gh-light-mode-only), is deprecated and will be removed in favor of the new method described above.

Could you update the other instances and replace the `Alternative text` with a simple summary of what is in the image?

**Review by ThunderDeliverer — CHANGES_REQUESTED** [review]

**Review by steven2308 — APPROVED** [review]
This was an interesting approach with images, but I don't think it was the wisest use of your time. For next occasions simply create a single version with a non transparent background and that should be enough.
Also, let's try to keep a PR limited to a single change, the new use case is kind of lost with all the image changes.

File: README.md [hunk 2]
```diff
@@ -207,7 +217,23 @@ After a small review Alice added them to her book. :book:
     }
 ```
 
-![alt text](images/MA_10.png)
+![alt text](images/dark_theme/MA_10.png#gh-dark-mode-only)
+![alt text](images/light_theme/MA_10.png#gh-light-mode-only)
+
+The last recipe was so delicious that Master added it to his cookbook also!
```
Comment (ThunderDeliverer) [inline README.md]: Just a minor suggestion:

```suggestion
The last recipe was so delicious that Master added it to his cookbook as well!
```

**Review by ThunderDeliverer — APPROVED** [review]
I'll merge this and apply changes in a separate commit, as I can't push to this PR 🦾

**Pull request merged**